test/karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function setKarmaConfig(config) {
         timeout: (process.env.CIRCLECI === 'true' ? 5 : 2) * 1000,
       },
     },
-    frameworks: ['mocha', 'webpack'],
+    frameworks: ['parallel', 'mocha', 'webpack'],
```

Comment:
Seems like it's getting [out of resources](https://app.circleci.com/pipelines/github/mui/mui-x/50037/workflows/9cc2ed86-d2bc-4dad-b2ca-0ac7c45e519e/jobs/286969), probably adding a concurrency [limit](https://github.com/joeljeske/karma-parallel#options) (like 8) will do, should still speedup by up to 8x, wdyt?